Tests/govuk_ios/govuk_ios_unit_tests/Specs/AppConfig/AppConfigProviderTests.swift

```diff
@@ -0,0 +1,51 @@
+@testable import govuk_ios
+import XCTest
+
+final class AppConfigProviderTests: XCTestCase {
+    var sut: AppConfigProvider?
```

Comment:
Can this be private?

Reply:
Hadn't noticed this had been merged before I commented so resolving comment and will make a note